src/CMakeLists.txt

```diff
@@ -3,7 +3,10 @@ add_subdirectory(3rd_party)
 include_directories(.)
 include_directories(3rd_party)
 include_directories(3rd_party/SQLiteCpp/include)
-include_directories(3rd_party/sentencepiece)
+# include_directories(3rd_party/sentencepiece)
```

Comment:
Is this meant to be commented out? I am not sure why `spm_encode` isn't found in CI on OS X, but I suspect this is the culprit.

Reply:
I don't think so. I've updated GitHub workflows recently, which may fix issues on macOS. I will rebase this branch with the current master and re-run builds.

Reply:
Thanks, @snukky! It still seems like there's something fishy happening on the spm front...